app/views/includes/map.pug

```diff
@@ -7,6 +7,6 @@ block content
 
   #map
 
-script(async defer src=`//maps.googleapis.com/maps/api/js?key=` + config.google.keys.restricted)
+script(src=`//maps.googleapis.com/maps/api/js?key=` + config.google.keys.restricted)
 script(src=`../../scripts-browserify/markerclusterer.js`)
-script(async defer src=`../../scripts-browserify/infobox.js`)
+script(src=`../../scripts-browserify/infobox.js`)
```

Comment:
Great! I experienced issues with this a few times since last we talked about this.